lib/mayaUsd/ufe/UsdUndoMaterialCommands.cpp

```diff
@@ -276,15 +344,8 @@ Ufe::SceneItem::Ptr UsdUndoAssignNewMaterialCommand::insertedChild() const
     return {};
 }
 
-std::string UsdUndoAssignNewMaterialCommand::resolvedMaterialScopeName()
-{
-    return UsdMayaJobExportArgs::GetDefaultMaterialsScopeName();
-}
```

Comment:
Not really needed anymore, since `UsdMayaJobExportArgs::GetDefaultMaterialsScopeName()` can simply be called directly.